README.md

```diff
@@ -48,15 +48,25 @@ The purpose is to build infrastructure in the field of large models, through the
 
 ### AI-Native Data App 
 ---
-- 🔥🔥🔥 [Released V0.5.0 | Develop native data applications through workflows and agents](https://docs.dbgpt.site/docs/changelog/Released_V0.5.0)
+- 🔥🔥🔥 [Released V0.6.0 | A set of significant upgrades](https://docs.dbgpt.cn/docs/changelog/Released_V0.6.0)
+  - [The AWEL upgrade to 2.0]()
```

Comment:
Is there a missing link URL?

Reply:
Yes, we will update relevant content in future blogs and ensure the link URL references the correct blog address. These updates will be included in a later PR.